bin/transfer_blurb_to_description.rb

```diff
@@ -33,6 +33,7 @@ def blurb
   end
 
   def metadata
+    # TODO: Update to parse TOML instead
     metadata_filename = File.join(path, 'metadata.yml')
     YAML.load_file(metadata_filename)
   end
```

Comment:
Anyone willing to take on this update?

I think we'd need to either run a `gem install` or `apt install` in the github ci file before this can be updated.

Good candidates from basic searching look like `toml-rb`, `tomlrb`, or `toml`.

Potentially another bikeshed??? See: [ruby gems](https://rubygems.org/search?query=toml) and [ubuntu packages](https://packages.ubuntu.com/search?keywords=ruby-toml&searchon=names)

Reply:
I've honestly never know about this script, and I don't think it's necessarily very useful. Anyone else ever use this? @SleeplessByte?

Reply:
> I've honestly never know about this script

See https://github.com/exercism/problem-specifications/issues/767 and https://github.com/exercism/problem-specifications/pull/775 for context. You knew about it 4 years ago :)

I don't know if we still need it.